doc/classes/float.xml

```diff
@@ -105,6 +118,7 @@
 			<argument index="0" name="right" type="Quat">
 			</argument>
 			<description>
+				Multiplies each component of the quaternion vector by the given [float].
```

Comment:
```suggestion
				Multiplies each component of the [Quat] by the given [float].
```

Since we have `[int]` elsewhere, I think it would be consistent to use `[Quat]` here.

I think "vector" is technically valid, but it can be confusing.

Reply:
Vector was a copy-paste mistake.